### DISCUSSION TITLE
The Christofides Algorithm

### DISCUSSION THREAD

**Discussion by giscus** [RealA10N/alon.kr#3] [General]
# The Christofides Algorithm

https://alon.kr/posts/christofides

**Comment by renjingneng** [comment]
thanks for illustrative article

**Reply by RealA10N** [reply]
Thank you for reading!

**Comment by RonGreenberg** [comment]
Very nice!
I learned this exact algorithm in college right about the time you posted this. Could've helped a lot, if only I knew... :sweat_smile:
